### PR TITLE
feat(web-iconics) : implement add blog page

### PR DESCRIPTION
**Pull request by Mekdeskebede**
In this branch add blog page is implemented
Screenshot 2023-04-24 164635
Screenshot 2023-04-24 164735

### CODE REVIEW

File: Web/starter_project_iconics/package.json [hunk 1]
```diff
@@ -29,6 +31,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.8.0",
+    "react-quill": "^2.0.0",
```
Comment (Aklile-Yilma) [inline Web/starter_project_iconics/package.json:34]: It's great that you built a functional UI, with WYSIWYG editor.

**Comment by Aklile-Yilma** [comment]
LGTM

**Review by Aklile-Yilma — DISMISSED** [review]
LGTM

**Review by sam5235 — DISMISSED** [review]

**Review by sam5235 — DISMISSED** [review]

**Review by sam5235 — CHANGES_REQUESTED** [review]
if you get stuck on the vercel error,create new branch(your branch is outdated), build, and rebase your branch.

**Pull request merged**